test/y2security/autoinst_profile/security_section_test.rb

```diff
@@ -18,36 +18,17 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 require_relative "../../test_helper"
-require "y2security/autoinst_profile"
+require "y2security/autoinst_profile/security_section"
 
 describe Y2Security::AutoinstProfile::SecuritySection do
-  let(:profile) { { "lsm" => { "select" => "selinux" } } }
-
   describe ".new_from_hashes" do
-    it "sets the lsm section" do
-      section = described_class.new_from_hashes(profile)
-      lsm = section.lsm
-      expect(lsm).to be_a(Y2Security::AutoinstProfile::LSMSection)
-      expect(lsm.select).to eq("selinux")
-      expect(lsm.parent).to eq(section)
-    end
-
-    context "when used the old 'selinux_mode' attribute" do
-      let(:profile) { { "selinux_mode" => "enforcing" } }
+    context "when only 'selinux_mode' attribute is given" do
+      let(:profile) { { "selinux_mode" => "enforcing", "lsm_select" => "selinux" } }
 
-      it "sets the selinux_mode attribute" do
+      it "sets the supported attributes" do
```

Comment:
NP: I'd say "it selects SELinux as LSM"

Reply:
I forget to remove the upper context. It does not select nothing just set the attributes as are defined, the select is done by the reader.